README.md

```diff
@@ -15,22 +15,42 @@ With the release of AndroidX, Android Support is now considered deprecated and w
 
 ## Binding Policies
 
-- This repository binds over 100 AndroidX libraries that are published to [NuGet.org](https://nuget.org). The full package list can be found in [config.json](config.json).
+- This repository binds over 200 (2022-09) AndroidX libraries that are published to [NuGet.org](https://nuget.org). The full 
+  package list can be found in [config.json](config.json).
 - AndroidX Java artifacts come from [Google's Maven Respository](https://maven.google.com/web/index.html#).
+- dependency Maven artifacts come from different maven repositories mostly:
```

Comment:
Maybe something like this to reduce duplication:

```
- AndroidX Java artifacts and some dependencies come from [Google's Maven Respository](https://maven.google.com/web/index.html#).
- Additional dependencies come from [Maven/Sonatype Central](https://repo1.maven.org/maven2/).
```

Reply:
changed